lib/rake_github/tasks/pull_requests/merge.rb

```diff
@@ -3,6 +3,8 @@
 require 'rake_factory'
 require 'octokit'
 
+require_relative '../../exceptions/no_pull_request_error'
```

Comment:
Requiring directly within the task in case it is required directly. Not sure of a good pattern for this...